docs/articles/documentation/extending-testcafe/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+title: Extending TestCafe
+permalink: /documentation/extending-testcafe/
+---
+# Extending TestCafe
+
+TestCafe is a powerful framework, however you may need a specific functionality that TestCafe does not provide initially.
```

Comment:
Well, I don't like it's non-normative tone I guess. Can we replace it with something like:
`TestCafe functionality can be extended using plugins` or smthg like that?